From 16a654836e45180f15eae9a3a7d7d9cb04e68303 Mon Sep 17 00:00:00 2001
From: Tapasweni Pathak <tapaswenipathak@gmail.com>
Date: Wed, 6 Mar 2019 23:46:16 +0530
Subject: [PATCH] drivers: gpu: drm: i915: Fix dead store

ret need not store return value from drm_dp_update_pauload_part1 as it
is not used after assignment.

---
 drivers/gpu/drm/i915/intel_dp_mst.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_dp_mst.c b/drivers/gpu/drm/i915/intel_dp_mst.c
index 4de247d..2c96e23 100644
--- a/drivers/gpu/drm/i915/intel_dp_mst.c
+++ b/drivers/gpu/drm/i915/intel_dp_mst.c
@@ -270,8 +270,6 @@ static void intel_mst_pre_enable_dp(struct intel_encoder *encoder,
 	temp = I915_READ(DP_TP_STATUS(port));
 	I915_WRITE(DP_TP_STATUS(port), temp);
 
-	ret = drm_dp_update_payload_part1(&intel_dp->mst_mgr);
-
 	intel_ddi_enable_pipe_clock(pipe_config);
 }
 
-- 
2.10.0

